libs/design/src/core/colorable/colorable.spec.ts

```diff
@@ -111,6 +111,7 @@ describe('colorInPalette', () => {
         expect(colorInPalette("white")).toEqual(true);
         expect(colorInPalette("primary")).toEqual(true);
         expect(colorInPalette("accent")).toEqual(true);
+        expect(colorInPalette("blue")).toEqual(true);
```

Comment:
I don't think we should allow arbitrary colors here. We allow white/black as those are special use-cases, but I think we should stick to a few theme colors.

Reply:
Same comment

> We didn't talk about this part yesterday. How do we style colors that are not $primary and $accent then? I still want to be able to add a `blue` variant to button, hero, callout.